.brazil.json

```diff
@@ -43,6 +43,7 @@
 
         "dynamodb": { "packageName": "AwsJavaSdk-DynamoDb" },
         "waf": { "packageName": "AwsJavaSdk-Waf" },
+        "migration-tool": { "packageName":  "AwsJavaSdk-MigrationTool"},
```

Comment:
The alternative I have in mind is `AwsJavaSdk-Tooling-Migration`. Feedback is welcome